src/odemis/gui/cont/tabs.py

```diff
@@ -3350,10 +3350,11 @@ def __init__(self, name, button, panel, main_frame, main_data):
 
                 if len(photods) > 1:
                     self._fbdet2 = photods[1]
-                    _, self._det2_cnt_ctrl = speccnt_spe.stream_panel.add_text_field("Detector 2", "")
+                    _, self._det2_cnt_ctrl = speccnt_spe.stream_panel.add_text_field("Detector 2", "", readonly=True)
```

Comment:
I was going to ask you to add this ;-)
Could you also add this line just after, to make the font less grey:
self._det2_cnt_ctrl.SetForegroundColour("#FFFFFF")

Reply:
done!